clients-mapreduce/src/main/java/uk/gov/gchq/palisade/clients/mapreduce/PalisadeRecordReader.java

```diff
@@ -51,6 +51,7 @@
  * In order to do this, we create a DataRequestResponse for each Resource and send it to the data service
  * created by the corresponding ConnectionDetail object.
  */
+@SuppressWarnings("java:S1135")
```

Comment:
Changes to other modules falls out of scope for this ticket

Reply:
Fixed